Formula/kontena-cli.rb

```diff
@@ -0,0 +1,72 @@
+class KontenaCli < Formula
+  desc "Command-line client for Kontena container & microservices platform"
+  homepage "https://kontena.io/"
+  url "https://github.com/kontena/kontena.git", :tag => "v1.3.4"
+  head "https://github.com/kontena/kontena.git"
+
+  bottle :unneeded
+
+  depends_on :ruby => "2.1"
+
+  def install
+    ruby_command = which("ruby")
+    gem_command = File.join(ruby_command.dirname, "gem")
```

Comment:
just use `system "gem", `

Reply:
Yes I think that was from the time when `which("ruby")` was not setting paths correctly.